mubench.reviewsite/tests/DatabaseTestCase.php

```diff
@@ -28,6 +28,7 @@ class DatabaseTestCase extends TestCase
         "result": "success",
         "runtime": 42.1,
         "number_of_findings": 23,
+        "-custom-stat-": "-stat-val-",
```

Comment:
Why do you need to touch the base test? What can we do to avoid changing the base test in case of an extension like this?